src/components/list/EntryList.js

```diff
@@ -11,8 +12,17 @@ import ListEntry from '../ui/ListEntry'
 import { TypeName } from '../ui/TypeName'
 import { ReactComponent as LeafIcon } from './leaf.svg'
 
-// TODO: use settings for this
-const convertMetersToMiles = (meters) => (meters * 0.000621371192).toFixed(2)
+const convertDistance = (distance, setting) => {
+  if (setting === 'imperial') {
+    return `${(distance * 0.000621371192).toFixed(2)} miles`
```

Comment:
Would making this floating point number a const be better? Just so people understand what this number is.